visualisation/visualise-raw-data/visualise-raw-data-functions.R

```diff
@@ -0,0 +1,72 @@
+plot_raw_data = function(national = TRUE,
+                         cutoff = 25){
+  
+  # Get observed data ------------------------------------------------------------------
+  source(here::here("utils", "get-us-data.R"))
+  
+  daily_deaths_state <- get_us_deaths(data = "daily") %>%
+    dplyr::mutate(day = ordered(weekdays(as.Date(date)), 
+                                levels=c("Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday")),
+                  epiweek_day = as.numeric(paste0(epiweek, ".", as.numeric(day)))) %>%
+    # filter out data from the last incomplete week
+    filter(epiweek_day < max(epiweek))
+  
+  
+  weekly_deaths_state <- daily_deaths_state %>%
+    dplyr::mutate(epiweek = lubridate::epiweek(date)) %>%
+    dplyr::group_by(state, epiweek) %>%
+    dplyr::summarise(deaths = sum(deaths),
```

Comment:
we think we are not going to have NAs?

Reply:
this is just to filter the last incomplete epiweek from the plot, not to filter out NAs

Reply:
its the sum. it will all be NA if a single day is NA.

Reply:
on a related note why is there a repeated call to epiweek here?